nepalingo-web/src/App.tsx

```diff
@@ -1,12 +1,36 @@
 import React from "react";
+import {
+  BrowserRouter as Router,
+  Route,
+  Routes,
+  Navigate,
+} from "react-router-dom";
+import User_auth from "./components/user_auth/User_auth";
+import Home from "./pages/Home/Home";
 import Header from "./components/Header";
+import supabase from "./components/user_auth/supabaseClient";
+
+const App: React.FC = () => {
+  const [isAuthenticated, setIsAuthenticated] = React.useState(false);
```

Comment:
you can import `useState` at the top so that you don't need to keep referencing it off of React. 

`import React, { useState } from 'react';` on line 1

Reply:
done